src/Main.java

```diff
@@ -1,9 +1,41 @@
+import java.util.ArrayList;
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
+//        Int Array
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
 
+//        String Array
+        String[] alphabetic = {"One", "Two", "Three", "Four", "Five", "Six", "", "Eight", "Nine", "Zero"};
 
+        boolean play = true;
+        String invalid = "Invalid Input";
+
+//        Makes the link to the Translator class getting the value in the Arrays; numeric & alphabetic
+
```

Comment:
Hoi Shea,
well done 👍
That was really organized and the logic is correct.
keep it up :)